pkg/proxy/handlers/spacelister_test.go

```diff
@@ -297,7 +297,7 @@ func workspaceFor(t *testing.T, fakeClient client.Client, name, role string, isH
 		commonproxy.WithNamespaces([]toolchainv1alpha1.SpaceNamespace{
 			{
 				Name: "john-dev",
-				Type: "default",
+				Type: toolchainv1alpha1.NamespaceTypeDefault,
```

Comment:
See https://github.com/codeready-toolchain/member-operator/pull/464#issuecomment-1646219312
```suggestion
				Type: "default",
```

Reply:
I just reverted the commit :) Thanks